apps/lockfile-explorer-web/src/helpers/displaySpecChanges.ts

```diff
@@ -0,0 +1,14 @@
+import { ISpecChange } from '../parsing/compareSpec';
+
+export const displaySpecChanges = (specChanges: Map<string, ISpecChange>, dep: string): string => {
+  switch (specChanges.get(dep)?.type) {
+    case 'add':
```

Comment:
Also, it's possible for the diff to be scrolled out of view, making it not obvious that there is any diff for this entry:
![image](https://user-images.githubusercontent.com/4673363/202327589-255220d1-fc24-4b2a-bd49-1ec8afbebc35.png)

Maybe we could provide a visual indicator, for example show `* package spec` instead of `package spec` if anything was modified?  Maybe we could even show a `*` in the listview as well:

![image](https://user-images.githubusercontent.com/4673363/202328021-996ba2da-9602-43b4-b176-a634b0c8b637.png)

I think these transformations are fairly important information when studying lockfiles, because they are the customizations that we have to fiddle with to fix problems.

Reply:
Added an asterisk to the "package spec *" header tab to indicate that there is a diff for the entry, I don't think we should show an asterisk in the list view because we need to load the package's package.json to be able to calculate if there is a spec change, (currently we just load the selected entry's package.json & calculate the spec changes then)